addons/point_of_sale/static/src/app/printer/render_service.js

```diff
@@ -95,15 +95,22 @@ const applyWhenMounted = async ({ el, container, callback }) => {
  */
 export const htmlToCanvas = async (el, options) => {
     el.classList.add(options.addClass || "");
-    // html2canvas expects the given element to be in the DOM
     return await applyWhenMounted({
         el,
         container: document.querySelector(".render-container"),
-        callback: async (el) =>
-            await html2canvas(el, {
-                height: Math.ceil(el.clientHeight),
-                width: Math.ceil(el.clientWidth),
-                scale: 1,
-            }),
+        callback: async (el) => {
+            return await new Promise((r) =>
+                htmlToImage
+                    .toCanvas(el, {
```

Comment:
Hello, a small question, why do you create new promise ? I think, you can directly do : 
```suggestion
            await htmlToImage
                    .toCanvas(el, {
```

Correct me if I'm wrong, but htmlToImage.toCanvas, send already a Promise, so I don't understand why you need to create a new one.

Also, why do you need to return the result ? it's used by the caller ?